DAL/Repositories/Implementaations/TodoRepositorie.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Microsoft.EntityFrameworkCore;
+using TodoAPI.DAL.DBContexts;
+using TodoAPI.DAL.Entity;
+using TodoAPI.DAL.Repositories.Interfaces;
+using TodoAPI.Extensions;
+using TodoAPI.Models.RequestViewModels;
+using TodoAPI.Models.ResponseViewModels;
+
+namespace TodoAPI.DAL.Repositories.Implementaations
+{
+    public class TodoRepositories : ITodoRepositorie
+    {
+        private readonly ApplicationDBContext _dbContext;
+
+        public TodoRepositories(ApplicationDBContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public async Task<TodoItem> CreateTaskAsync(RequestModel todoItem)
+        {
+
+            TodoItem requestbody = new TodoItem
+            {
+                ID = todoItem.Id,
+                Name = todoItem.Name,
+                Description = todoItem.Description,
+                CreatedAt = DateTime.Now,
+            };
+
+            await _dbContext.AddAsync(requestbody);
+            await _dbContext.SaveChangesAsync();
+
+            return requestbody;
+        }
+
+        public async Task<string> DeleteTaskByIdDbAsync(int id)
```

Comment:
Rename it to DeleteTaskAsync, because in general we delete an entry from db based on id and if you don't have multiple delete methods you can name is simple DeleteTaskAsync.